gym_snake/envs/snakeGameGym.py

```diff
@@ -18,10 +18,18 @@ class SnakeGameGym(SnakeGame):
 	Inherits the SankeGame class that runs the Snake Game.
 	"""
 
-	def __init__(self, fps: int, use_pygame: bool = True):
+	def __init__(self,
+		board_height: int, 
+		board_width:int, 
+		use_pygame: bool):
```

Comment:
Can you add default values for all of these arguments throughout the different functions you changed? I think it makes it easier for people who are new to the codebase to get started easily if everything has a default. 

```suggestion
		use_pygame: bool=False):
```